commands/Tools/texttobinary.js

```diff
@@ -0,0 +1,16 @@
+const { Command } = require('klasa');
+
+module.exports = class Binary extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			description: 'Convert Discord Text to Binary!',
+			usage: '<text:str>'
+		});
+	}
+
+	async run(msg, [string]) {
+		return msg.send(`\`\`\`\n${string.split('').map(char => char.charCodeAt(0).toString(2)).join(' ')}\n\`\`\``);
```

Comment:
1. Use klasa's util.codeBlock, it'll improve readability.
1. You may want to use `.padStart(8, '0')` after `.toString(2)` so if it outputs `101`, it gets formatted as `00000101`.